lib/simplifyExpression/arithmeticSearch/index.js

```diff
@@ -2,35 +2,40 @@ const ChangeTypes = require('../../ChangeTypes');
 const evaluate = require('../../util/evaluate');
 const Node = require('../../node');
 const TreeSearch = require('../../TreeSearch');
+const {build, query} = require('math-nodes');
 
 // Searches through the tree, prioritizing deeper nodes, and evaluates
 // arithmetic (e.g. 2+2 or 3*5*2) on an operation node if possible.
 // Returns a Node.Status object.
 const search = TreeSearch.postOrder(arithmetic);
 
+
+const isIntegerFraction = (node) => query.isNeg(node) ? isIntegerFraction(node.args[0]) : query.isDiv(node) && node.args.every(arg => Number.isInteger(query.getValue(arg)));
```

Comment:
having a helper like this would be helpful yeah - I thought we already had an `isIntegerFraction` exported from somewhere, but yeah that should live somewhere that can be used in multiple places